src/common/ipc.ts

```
@@ -61,6 +61,9 @@ export interface IpcBroadcastParams<A extends any[] = any> {
 }
 
 export function broadcastIpc({ channel, frameId, frameOnly, webContentId, filter, args = [] }: IpcBroadcastParams) {
+  if (!webContents) {
+    return
```

Why this is needed? This method intended to work only in main process and calls from renderer must not happen.

---

I can take it away but function name does not really tell that this is only from main->renderer (given that this locates under common).

---

Maybe rename to `broadcastToViews`? 
Or maybe `webContents = webContents || remote.webContents` if this make it work from renderer?

---

Anyway, silently ignore call (even unexpected) from renderer doesn't sound right IMHO..

---

I'll revert this change, out-of-scope for this PR.